openedx/features/enterprise_support/api.py

```diff
@@ -248,7 +248,7 @@ def enterprise_customer_for_request(request, tpa_hint=None):
         except EnterpriseCustomer.DoesNotExist:
             pass
 
-    ec_uuid = request.GET.get('enterprise_customer')
+    ec_uuid = request.GET.get('enterprise_customer') or get_enterprise_customer_uuid_from_cookie(request)
```

Comment:
Or you could make the wrapper function return the result of this line and the EnterpriseCustomer lookup below. Something like `get_enterprise customer_for_request`.